### PR TITLE
Initial support for single json config

### PR DESCRIPTION
**Pull request by rafaelfolco**
crucible run --from-file <config.json>

### CODE REVIEW

File: bin/crucible-run.json.example [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    { "arg": "bs", "vals": [ "4K", "16K" ] }
+                ]
+            }
+        ]
+    },
+    "tool-params": {
+    },
+    "passthru-args": ""
+}
+
```
Comment (k-rister) [inline bin/crucible-run.json.example:31]: The `bin` directory seems like an odd location for this file.

File: bin/get_config.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
```
Comment (k-rister) [inline bin/get_config.py:1]: I'm thinking that this belongs in https://github.com/perftool-incubator/toolbox/tree/main/bin, right?  Eventually we are going want to be able to extract things from this configuration file in a bunch of different places and putting a tool like this in toolbox is the best way to ensure it's available.

We should also maybe use a slighly more descriptive name, `get_config.py` is pretty vague.

File: bin/crucible-run.json.example (hunk 1 above)
Comment (rafaelfolco) [reply bin/crucible-run.json.example:31]: ack

File: bin/get_config.py (hunk 2 above)
Comment (rafaelfolco) [reply bin/get_config.py:1]: agreed

**Review by k-rister — APPROVED** [review]

**Comment by k-rister** [comment]
We should start thinking about creating test cases for this code in [`crucible-ci`](https://github.com/perftool-incubator/crucible-ci).

**Comment by rafaelfolco** [comment]
> We should start thinking about creating test cases for this code in [`crucible-ci`](https://github.com/perftool-incubator/crucible-ci).

Indeed. This should not break anything, we're just adding a new option, not changing the current workflows.

**Pull request merged**